README.md

```diff
@@ -1,2 +1,451 @@
+    "createdAt": "2016-01-21 10:45:01",
+    "updatedAt": "2016-01-21 10:45:01",
+    "updatedByUser": "tanya"
+  },
+  {
+    "_id": 5,
+    "section": "Frequently Asked Questions: Volunteers",
+    "label": "faq-volunteers",
+    "content": [
+      {
+        "question": "How much time should I dedicate?",
+        "answer": "Culpa tousled semiotics pop-up, esse DIY pinterest sapiente fixie lomo irure VHS lumbersexual kinfolk +1. Drinking vinegar cold-pressed slow-carb lo-fi. Photo booth fanny pack semiotics, laboris 8-bit migas pinterest vegan distillery kale chips."
+      },
+      {
+        "question": "Is this program restricted to only women?",
+        "answer": "Banh mi enim you probably haven't heard of them tattooed. Duis mlkshk mollit, banh mi veniam occupy wayfarers fixie fingerstache sed ad. Plaid salvia hella, affogato organic kickstarter 3 wolf moon est godard nulla pour-over excepteur quinoa sapiente."
+      }
+    ],
+    "createdAt": "2016-01-21 10:45:01",
+    "updatedAt": "2016-01-21 10:45:01",
+    "updatedByUser": "tanya"
+  },
+  {
+    "_id": 6,
+    "section": "Frequently Asked Questions: Non-Profits",
+    "label": "faq-non-profits",
+    "content": [
+      {
+        "question": "When will development on my project begin and finish? We would like to go live in 3 months. Is that possible?",
+        "answer": "Chartreuse elit PBR&B, reprehenderit sint readymade marfa accusamus keytar. Tattooed yuccie green juice, schlitz humblebrag marfa portland excepteur locavore eu ullamco knausgaard sustainable."
+      }
+    ],
+    "createdAt": "2016-01-21 10:45:01",
+    "updatedAt": "2016-01-21 10:45:01",
+    "updatedByUser": "tanya"
+  }
+
+}
+
+```
+
+
+
+### Team Members
+
+Open: `GET | GET :id/label`
+
+Secure: `POST | PUT :id/label`
+
+#### GET api/content/team_members/
+
+Returns a list of team members content.
+
+##### parameters
+
+| Name | Description | Data Type |
+| ---- | ----------- | --------- |
+| active  | *(optional)* active status | boolean |
+| role  | *(optional)* founder, board member | string |
+
+
+##### example response
+
+```
+{
+  {
+    "_id": 1,
+    "label": "vrathi",
+    "name": {
+      first: "Vinita",
+      last: "Rathi"
+    },
+    "role": "founder",
+    "active": 1,
+    "dateJoined": "2015-04-21",
+    "createdAt": "2016-01-21 10:45:01",
+    "updatedAt": "2016-01-21 10:45:01",
+    "updatedByUser": "tanya"
+  },
+  {
+    "_id": 2,
+    "label": "lkastilio",
+    "name": {
+      first: "Lili",
+      last: "Kastilio"
+    },
+    "role": "board member",
+    "board": "Volunteer Onboarding",
+    "active": 1,
+    "dateJoined": "2015-04-21",
+    "createdAt": "2016-01-21 10:45:01",
+    "updatedAt": "2016-01-21 10:45:01",
+    "updatedByUser": "tanya"
+  }
+}
+
+```
+
+
+### Volunteers
```

Comment:
The board member and volunteer info here is limited to what we need to output in website content, which is why it made sense to segregate them in this content API. The volunteer-focussed API will contain additional details more in line with volunteer management in the Volunteer/Project portal. Discussed with Vinita/Dionne et al so we're all on the same page about that. Also, that it's highly likely we can integrate the Portal API in the future so some of these details _could_ come from there eventually.